lib/screens/home.dart

```diff
@@ -164,7 +164,10 @@ class HomeScreen extends StatefulWidget {
 
 class _HomeScreenState extends State<HomeScreen> with TickerProviderStateMixin {
   final _scaffoldKey = GlobalKey<ScaffoldState>();
-
+  late StreamingSharedPreferences streamPrefs;
+  String ttsChannel = '';
+  bool isTtsActive = false;
```

Comment:
This should be a getter https://flutterbyexample.com/lesson/getters-and-setters it should be true if `ttsChannel` is not set.

Reply:
@kevmo314 can you take a look at this and see if this is the correct implementation of the getter here?